apis/controllers/storedPinController.js

```diff
@@ -0,0 +1,24 @@
+const { storedPinService } = require('../services');
+const { catchAsync } = require('../../utils/error');
+
+const storePin = catchAsync(async (req, res) => {
+  const userId = req.headers.userid;
+  const pinId = req.params.pinId;
+
+  await storedPinService.storePin(userId, pinId);
+
+  res.status(201).send({ message: 'pin stored sucessfully' });
+});
+
+const deleteStoredPin = catchAsync(async (req, res) => {
+  const userId = req.headers.userid;
+  const pinId = req.params.pinId;
+
+  await storedPinService.deleteStoredPin(userId, pinId);
+
+  res.status(200).send({ message: 'pin deleted sucessfully' });
+});
+module.exports = {
+  storePin,
+  deleteStoredPin,
+};
```

Comment:
```suggestion
  res.status(200).send({ message: 'DELETE PIN SUCCESS' });
});

module.exports = {
  storePin,
  deleteStoredPin,
};
```
client에게 response 되는 메세지는 가급적 대문자로 작성해주시고, 가독성을 위해 exports  하는 부분은 한 줄 띄워서 작성해주세요 :)

Reply:
감사합니다 수정하였습니다.